packages/terra-modal-manager/src/terra-dev-site/test/modal-manager/DisclosureComponent.jsx

```diff
@@ -132,8 +122,8 @@ class DisclosureComponent extends React.Component {
           {' '}
           {identifier}
         </p>
-        {disclosureManager && disclosureManager.releaseFocus ? <p>Modal has lost focus!</p> : null }
-        {disclosureManager && disclosureManager.requestFocus ? <p>Modal has gained focus!</p> : null }
+        {document.activeElement !== document.querySelector('[role="dialog"][aria-modal="true"]') ? <p>Modal has lost focus!</p> : null}
```

Comment:
You might just remove these altogether. With these changes, we shouldn't need to communicate any focus-based logic to our consumers.

Reply:
Updated in these commits:
* https://github.com/cerner/terra-framework/pull/608/commits/617f82794ed734957f05f309302587c17b8c6de9
* https://github.com/cerner/terra-framework/pull/608/commits/e42cc17870dc7c16e7dafc4c981f131b75d72c08